README.md

```diff
@@ -221,6 +367,26 @@ configure(project(':itest')) {
 }
 ```
 
+<details>
```

Comment:
Could you also change `compile` to `implementation` to the related Groovy snippet for consistency?

Reply:
Done